lib/butler/plugins/cowsay.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Butler.Plugins.Cowsay do
+  use Butler.Plugin
```

Comment:
As far as I know the plugins should be able to reference the main butler repo and then pull in just the `Butler.Plugin` module. If thats not the case we should try to figure out why.